CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ option(BUILD_TEXT_INPUT_PLUGIN "Include the text input plugin in the finished bi
 option(BUILD_RAW_KEYBOARD_PLUGIN "Include the raw keyboard plugin in the finished binary. Enables raw keycode listening in flutter via the flutter RawKeyboard interface." ON)
 option(BUILD_TEST_PLUGIN "Include the test plugin in the finished binary. Allows testing platform channel communication." OFF)
 option(BUILD_OMXPLAYER_VIDEO_PLAYER_PLUGIN "Include the omxplayer_video_player plugin in the finished binary. Allows for hardware accelerated video playback in flutter using omxplayer." ON)
-option(BUILD_GSTREAMER_VIDEO_PLAYER_PLUGIN "Include the gstreamer_video_player plugin in the finished binary. Allows for more stable, hardware accelerated video playback in flutter using gstreamer." ON)
-option(TRY_BUILD_GSTREAMER_VIDEO_PLAYER_PLUGIN "Don't throw an error if the gstreamer libs aren't found, instead just don't build the gstreamer video player plugin in that case." ON)
+option(BUILD_GSTREAMER_AUDIO_VIDEO_PLAYER_PLUGIN "Include the gstreamer based video and audio plugins in the finished binary. Allows for more stable, hardware accelerated video playback in flutter using gstreamer." ON)
+option(TRY_BUILD_GSTREAMER_AUDIO_VIDEO_PLAYER_PLUGIN "Don't throw an error if the gstreamer libs aren't found, instead just don't build the gstreamer audio and video player plugin in that case." ON)
```

Comment:
So we don't break meta-flutter, we should add a separate build option for the audioplayer plugin.